src/views/Map/MainContent.js

```diff
@@ -19,6 +19,8 @@ import {Modal, TextField} from '@mui/material';
 import Typography from '@mui/material/Typography';
 import Button from '@mui/material/Button';
 import ModalInfo from '../../components/ModalInfo';
+import red from '@mui/material/colors/red';
+import blue  from '@mui/material/colors/blue';
 
 const inputContainerStyles = {
   position: 'absolute',
```

Comment:
Molaría que lo hiciéramos con display: 'flex' que es mucho más fácil cuando lo conoces y más moderno. Te paso un par de links pero si no lo conoces, te hago una pequeña introducción.
[https://css-tricks.com/snippets/css/a-guide-to-flexbox/](url)
yo aprendí con este juego [https://flexboxfroggy.com/#es](url) hazlo si no lo conoces que es muy educativo